docs/technical-doc/dummy-network-generation.rst

```diff
@@ -45,9 +45,10 @@ Setup your node to use the secret you just generated as its public key and staki
 
     {"secret_key":"SECRETK","public_key":"PUBK"}
 
- * modify the file `massa-node/base_config/initial_ledger.json` :
+ * clear and modify the file `massa-node/base_config/initial_ledger.json` :
 
  .. code-block:: javascript
+
```

Comment:
Can you edit this to have the actual format please ?

Reply:
Done